Automattic/newspack-blocks
# feat: homepage posts patterns with ad unit

### All Submissions:






### Changes proposed in this Pull Request:

Updates Homepage Posts patterns 2, 7, 14, 26 and implements new pattern 31 as described in https://github.com/Automattic/newspack-ads/issues/210.

This PR also inaugurates "placeholder blocks", a simple config to allow a more descriptive missing block. The placeholder should provide an easy button to remove the missing block, a description and a link to the plugin that provides the block:

<img width="432" alt="image" src="https://user-images.githubusercontent.com/820752/173669408-29dee800-b978-4f2e-b0b7-548b4381e7ef.png">



Closes https://github.com/Automattic/newspack-ads/issues/210

### How to test the changes in this Pull Request:

1. With Newspack Ads active, create a new page and add patterns listed above
2. Confirm they reflect https://github.com/Automattic/newspack-ads/issues/210
3. Deactivate Newspack Ads
4. Refresh the editor
5. Confirm the Ad Unit blocks are replaced with the placeholder as pictured above

### Other information:
